doc/news/changes/minor/20171118LucaHeltai

```diff
@@ -1,4 +1,4 @@
-New:Added Utilities::pack and Utilities::unpack, to serialize and unserialize arbitrary 
```

Comment:
supposedly you can write `Add Utilities...`.

Reply:
Good point -- fixed.